### PR TITLE
webpack

### PR DESCRIPTION
**Pull request by xoron**

### CODE REVIEW

File: src/components/PageComponents/Messages/Message.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const Message = ({
             message.state === "ack" ? "text-textPrimary" : "text-textSecondary"
           }`}
         >
-          {message.data}
+          {message.data}asdasdkjahkd!!!!!!!123!!!!!!!!
```
Comment (PixnBits) [inline src/components/PageComponents/Messages/Message.tsx:70]: Probably not intentionally committed?

**Comment by PixnBits** [comment]
Just a fellow user, but I'm curious what the purpose of adding webpack is? Is it related to the `ModuleFederationPlugin`?
(Would be useful to fill out the PR description? :pray: )

**Comment by xoron** [comment]
Sorry. Was meant for this to be a pr to my own fork will close pr.

Yes, I want to add module federation.

File: src/components/PageComponents/Messages/Message.tsx (hunk 1 above)
Comment (xoron) [reply src/components/PageComponents/Messages/Message.tsx:70]: Sorry. I'm was trying out changes on my own fork. This isn't to be merged 😅

**Pull request closed without merging**